src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -389,52 +397,160 @@ sub-<label>/[ses-<label>/]
-[`acq-<label>`](../99-appendices/09-entities.md#acq)
+If more than one run of the same acquisition and direction has been acquired, the
+[`run-<index>`](../99-appendices/09-entities.md#run) key/value pair MUST be used:
+`_run-1`, `_run-2`, `_run-3` (and so forth.)
+When there is only one scan of a given acquisition and direction, the run key MAY be
+omitted.
+The [`run-<index>`](../99-appendices/09-entities.md#run) key/value pair is RECOMMENDED
+to encode the splits of multipart DWI scans (see [below](#multipart-split-dwi-schemes).)
+
+The OPTIONAL [`acq-<label>`](../99-appendices/09-entities.md#acq)
+key/value pair corresponds to a custom label the user may use to
+distinguish different sets of parameters.
+
+The OPTIONAL [`dir-<label>`](../99-appendices/09-entities.md#dir)
 key/value pair corresponds to a custom label the user may use to
-distinguish different set of parameters. For example this should be used when a
-study includes two diffusion images - one single band and one multiband. In such
-case two files could have the following names:
-`sub-01_acq-singleband_dwi.nii.gz` and `sub-01_acq-multiband_dwi.nii.gz`,
-however the user is free to choose any other label than `singleband` and
-`multiband` as long as they are consistent across subjects and sessions. For
-multiband acquisitions, one can also save the single-band reference image as
-type `sbref` (for example, `dwi/sub-control01_sbref.nii[.gz]`) The bvec and bval files
-are in the [FSL format](https://fsl.fmrib.ox.ac.uk/fsl/fslwiki/FDT/UserGuide#DTIFIT):
-The bvec files contain 3 rows with n space-delimited floating-point numbers
-(corresponding to the n volumes in the relevant NIfTI file). The first row
-contains the x elements, the second row contains the y elements and third row
-contains the z elements of a unit vector in the direction of the applied
-diffusion gradient, where the i-th elements in each row correspond together to
-the i-th volume with `[0,0,0]` for non-diffusion-weighted volumes. Inherent to
-the FSL format for bvec specification is the fact that the coordinate system of
-the bvecs is with respect to the participant, defined by the axes of the
-corresponding `dwi.nii` file, and not the magnet's coordinate system.
-Thus, any rotations applied to `dwi.nii` also need to be applied to the
-corresponding bvec file.
-
-bvec example:
+distinguish different sets of phase-encoding directions.
+
+**Combining multi- and single-band acquisitions**.
+The single-band reference image MAY be stored with suffix `sbref` (for example,
+`dwi/sub-control01_sbref.nii[.gz]`,) as long as the image has no corresponding
+[gradient information (`.bval` and `.bvec` sidecar files)](#required-gradient-orientation-information)
+to be stored.
+
+Otherwise, if some gradient information is associated to the single-band diffusion
+image and a multi-band diffusion image also exists, the `acq-<label>` key/value pair
+MUST be used to distinguish both images.
```

Comment:
I'm assuming so, but just to confirm, does this match current practice? This seems to be a new explicit requirement, although I see that there's never been `_sbref.bval`.

Reply:
If you see a `_sbref.bval` file then it should be filled with zeros. There could be a use case for nonzero (but very low b-vals << 1000) but why don't use `_dwi` which is more appropriate?

This does match current practice, and as you were pointing, it was implicit in the spec. I just made it explicit, if that makes sense.

Reply:
There's one large case, which is siemens scanners can't make a b=0 only series without  b > 0 scan included. So the sbref will include a few b=0 images followed by a random b=1000. +1 for optional gradient files for sbrefs

Reply:
I think the BIDS curator has two options in that case:

- Store a `_dwi` file, although there are not enough orientations to fit a diffusion model this is indeed a *diffusion-weighted* MRI.
- During conversion, drop the b > 0 volume to store a `_sbref` file.

Allowing `_sbref.bval` adds quite a bit of complexity to PyBIDS and software and irregularity (w.r.t. sbrefs under `func/`), while IMHO misrepresenting the nature of the file.

Would you reconsider your +1 to having optional gradient files for sbrefs?

Reply:
Dropping the b > 0 images would be a processing step, so the sbrefs would no longer be "raw" like the other images. This is actually the same issue that the reverse phase encoding direction images have - the `fmaps/*_epi` images require .bval and .bvec files to be interpreted correctly for distortion correction. 

These are major BIDS limitations that are prevent dmri workflows from using real BIDS and require users put the extra gradient information files into .bidsignore. I think there is another PR open for this, though. I'd be happy if there was even a sidecar field that has the bvals in it, since the gradient directions are irrelevant fir sbrefs and epi fieldmaps.

Reply:
> Dropping the b > 0 images would be a processing step, so the sbrefs would no longer be "raw" like the other images.

Dropping the b > 0 would be the same as people are actually doing with nonsteady states in functional MRI (at some places), for instance. Or if you intendedly leave out of conversion a whole bunch of acquisitions that are otherwise already in the DICOM structure. I'm not saying anyone should do this, but it is an option - this is outside of BIDS purview.

>  This is actually the same issue that the reverse phase encoding direction images have - the fmaps/*_epi images require .bval and .bvec files to be interpreted correctly for distortion correction.

I don't think this is the same issue. If you have diffusion weights (i.e., nonzero b-values) then you are indeed acquiring DWI - and these files should not be under `fmap/` regardless of whether you can fit a diffusion model or not. I *think* BIDS has sufficient specifications to write these files under `dwi/` as `_dwi.nii` files with corresponding gradient info. 

The problem of estimating a fieldmap using these DWIs is a completely different issue, which is actually addressed in #622.

Finally, accepting `sbref.bval` files would probably not be backward compatible (and a big change I would leave for another PR to keep this one with a focused scope).

> These are major BIDS limitations that are prevent dmri workflows from using real BIDS and require users put the extra gradient information files into .bidsignore.

Can you point at some publicly shared dataset?